<p>For readability reasons, the same starting token should be used for each comment line.</p>
<p>Rule can be configured to use either '!' or '#'</p>

<h2>Noncompliant Code Example</h2>
With token "#":
<pre>
! My comments...
! My comments...
</pre>

<h2>Compliant Solution</h2>
With token "#":
<pre>
# My comments...
# My comments...
</pre>



