<p>For readability reasons, all properties and comments should be aligned at column 1.</p>

<h2>Noncompliant Code Example</h2>
<pre>
           key1=value1
      key2=value2

        # comments
        key3=value3
</pre>

<h2>Compliant Solution</h2>
<pre>
key1=value1
key2=value2

# comments
key3=value3
</pre>

