<p>
    Java 9 expects properties file to be encoded in UTF-8 instead of ISO-8859-1. Even if Java 9 provides some fallback
    mechanisms to ISO-8859-1 while loading properties, in some corner cases, you might face unexpected behaviors for
    ISO-8859-1 characters not matching UTF-8 characters (meaning characters whose code points are over U+007F). For
    instance, instead of <code>Jörg</code>, <code>J�rg</code> might be displayed. To make sure to avoid any display
    issue, either:
</p>
<ul>
    <li>Escape all characters whose code points are over U+007F with Unicode escapes (<code>\uXXXX</code>)</li>
    <li>Or explicitly load properties files with ISO-8859-1 encoding</li>
</ul>
<p>
    This rule applies only when `sonar.jproperties.sourceEncoding` is set to `ISO-8859-1' (default value) and it raises
    an issue each time a character whose code point is over U+007F is found.
</p>


<h2>Noncompliant Code Example</h2>
<pre>
my.name: Jörg
</pre>

<h2>Compliant Solution</h2>
<pre>
my.name: J\u00f6rg
</pre>

<h2>See</h2>
<ul>
    <li><a target="_blank"
           href="https://docs.oracle.com/javase/9/intl/internationalization-enhancements-jdk-9.htm#JSINT-GUID-5ED91AA9-B2E3-4E05-8E99-6A009D2B36AF">Oracle
        - Internationalization Enhancements in JDK 9</a></li>
    <li><a target="_blank" href="http://openjdk.java.net/jeps/226">OpenJDK - JEP 226: UTF-8 Property Files</a></li>
</ul>
