<p>Shared coding conventions allow teams to collaborate efficiently. This rule checks that all keys match a provided regular expression.</p>
<h2>Noncompliant Code Example</h2>
With format "^[a-z]+[a-z0-9_\-\.]*$":
<pre>
myKey=abc
</pre>
<h2>Compliant Solution</h2>
With format "^[a-z]+[a-z0-9_\-\.]*$":
<pre>
my.key=abc
</pre>

