<p>Having to scroll horizontally makes it harder to get a quick overview and understanding of any piece of code.</p>
<h2>Noncompliant Code Example</h2>
With maximum length set to 50:
<pre>
myProperty=blabla blabla blabla blabla blabla blabla blabla blabla blabla
</pre>
<h2>Compliant Solution</h2>
With maximum length set to 50:
<pre>
myProperty=blabla blabla blabla blabla blabla\
           blabla blabla blabla blabla
</pre>

