<p>
    This rule raises an issue each time a translation key is defined in a locale resource bundle but not in the default
    resource bundle.
</p>

<h2>Noncompliant Code Example</h2>

<i>messages.properties</i>
<pre>
# Noncompliant: "farewell" property is missing
greetings: Hello
inquiry: How are you?
</pre>

<i>message_fr_FR.properties</i>
<pre>
greetings: Bonjour
farewell: Au revoir
inquiry: Comment allez-vous ?
</pre>

<h2>Compliant Solution</h2>

<i>messages.properties</i>
<pre>
greetings: Hello
farewell: Goodbye
inquiry: How are you?
</pre>

<i>message_fr_FR.properties</i>
<pre>
greetings: Bonjour
farewell: Au revoir
inquiry: Comment allez-vous ?
</pre>

<h2>See</h2>
<ul>
    <li>Related rule {rule:jproperties:missing-translations}</li>
</ul>
