<p>For readability reasons, the same separator should be used all the time.</p>
<p>Rule can be configured to check one of the following formats:
<ul>
    <li>key=element</li>
    <li>key:*whitespace*element</li>
</ul>
</p>
<h2>Noncompliant Code Example</h2>
With format "key:*whitespace*element":
<pre>
key1=value1
key2 : value2
key3 :value3
key4:    value4
</pre>
<h2>Compliant Solution</h2>
With format "key:*whitespace*element":
<pre>
key1: value1
key2: value2
key3: value3
key4: value4
</pre>



